CONTRIBUTING.md

```diff
@@ -7,14 +7,19 @@
 
 ## 🏡 Setup Repository
 
-- ### Step 1: Clone Repo
+- ### Step 1: Fork the Repository
+  To contribute to this project, you'll need to fork the repository to your own GitHub account. Here's how:
+  1. Click the "Fork" button in the top right corner of this GitHub repository page.
+  2. A copy of this repository will be created in your own GitHub account.
```

Comment:
```suggestion

You can click "fork" button in top right corner to create a copy of this repo in your account where changes can be made.
```

Can we shorten this to 1 line?

Reply:
Sure, one line sounds good to me! Thank you for the suggestion.